.github/workflows/validate.yml

```
@@ -13,11 +13,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v4
-    - uses: actions/setup-node@v4
-      with:
-        node-version: 16
-    - name: Check for broken internal links
-      run: |
-        npm i
-        npm run-script check-links-local
+      - name: Checkout Current Repo
+        uses: actions/checkout@v4
+      - name: Checkout Actions
+        uses: actions/checkout@v4
+        with:
+          repository: hazelcast/hazelcast-docs
+          sparse-checkout: |
+            .github/actions/validate/action.yml
```

```suggestion
            .github/actions/validate
```

im not exactly sure what the syntax is here - does it work as is?